client/src/components/AskQuestion.jsx

```diff
@@ -33,28 +84,79 @@ class AskQuestion extends React.Component {
       question: event.target.value
     });
   }
-  handleSubmit() {
+  handleSubmit(question) {
     event.preventDefault();
-    this.props.addQuestion(this.state.question);
+    this.props.addQuestion(question);
     this.reinitializeState();
   }
   reinitializeState() {
     this.setState({
       question: ''
     })
   }
+  showModal() {
```

Comment:
if you wanted, you could make this into just one toggleModal function that just changes showModal to be !this.state.showModal (the opposite)

Reply:
Good call